recipes/greenlet/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "greenlet" %}
+{% set version = "0.4.12" %}
+{% set sha256 = "e4c99c6010a5d153d481fdaf63b8a0782825c0721506d880403a3b9b82ae347e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - greenlet
+
+about:
+  home: https://github.com/python-greenlet/greenlet
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Lightweight in-process concurrent programming'
+
+  description: |
```

Comment:
You should replace the ticks and doublequotes in the description with ASCII quotes and doublequotes. This fixes the code problem with py35/py36 appveyor builds